src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/StringMetadata.php

```diff
@@ -53,6 +63,10 @@ public function toJsonSchema(): array
             $jsonSchema['pattern'] = $this->pattern;
         }
 
+        if (null !== $this->format) {
+            $jsonSchema['format'] = $this->format;
```

Comment:
this is not tested, right?

Reply:
The whole class is not tested, but I'll write a test for it